Umee-Bots/Large-Deposit-Withdrawal/src/utils.ts

```diff
@@ -0,0 +1,46 @@
+import BigNumber from "bignumber.js";
+import { ethers, Finding, FindingSeverity, FindingType, LogDescription } from "forta-agent";
+

```

Comment:
```suggestion

BigNumber.set({ DECIMAL_PLACES: 18 });

```
This is just to avoid too many decimal places when making `BigNumber` operations.